apps/cookbook/src/app/examples/page-example/fixed-footer-tabs/fixed-footer-tabs-example.component.ts

```diff
@@ -8,27 +8,24 @@ const template = `<!-- This container is only needed due to layout of larger scr
   </div>
 
   <kirby-tab-bar>
-    <kirby-tab-button routerLink="dashboard">
-      <kirby-icon name="person-outline"></kirby-icon>
-      <kirby-icon name="person" selected-tab></kirby-icon>
-      Dashboard
+    <kirby-tab-button routerLink="overview">
+      <kirby-icon name="kirby"></kirby-icon>
```

Comment:
I just went to the documentation for icon to see what icon `kirby` is. I noticed that it's the exact same icon as `overview-outline`. I propose that `overview-outline` is used instead, as that name is more descriptive, and that we create an issue on removing the icon named `kirby`, so we don't have any duplicates :)

Reply:
Good catch @mark-drastrup ! 👌🏻 🔍 
I've updated the docs example to `overview-outline`: https://github.com/kirbydesign/designsystem/pull/3126/commits/e8ee13e34ac6a5e6cac4af4eb701c8971d218074

Regarding your 2nd comment: We shouldn't do that 😅 
We use the Kirby icon everywhere for examples - `overview` and `overview-outline` currently is the same icon, but that could potentially change later...

Reply:
@jakobe Ahh I see 😅 Yeah let's keep it the way it is then 😄